cmd/emailreport/main.go

```diff
@@ -0,0 +1,112 @@
+/// Copyight (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/BurntSushi/toml"
```

Comment:
similar to the other file, I think we need to re-order this. I know `gofmt` probably sorted the list for you, but I think that is because `gofmt` will sort on a 'block' basis, and we had not broken this into blocks with a blank line. Hence, I think if you try:
```
import (
    "flag" ....
    <blank line>
    "github.com/BurntSushi/toml"
  .....
)
```

Then it should work...